modelbuilders_bench/xgb_mb.py

```diff
@@ -180,8 +188,14 @@ def shap_accuracy(new, ref):
     "enable_experimental_json_serialization": params.enable_experimental_json_serialization,
 }
 
-if params.threads != -1:
+if params.threads == -1:
+    # SHAP value calculation is faster with using logical cores as number of threads
+    import psutil
+
+    daal4py.daalinit(psutil.cpu_count())
```

Comment:
If number of physical cores is preferred, `cpu_count(logical=False)` is correct one. Otherwise, `multiprocessing.cpu_count` can give same number.